packages/react-examples/src/react/PeoplePicker/docs/PeoplePickerBestPractices.md

```diff
@@ -5,8 +5,15 @@
 
 ### Accessibility
 
+#### Mobile Accessibility
+
 PeoplePicker dropdowns render in their own layer by default to ensure they are not clipped by containers with `overflow: hidden` or `overflow: scroll`. This causes extra difficulty for people who use touch-based screen readers, so we recommend rendering the PeoplePicker inline unless it is in an overflow container. To do so, set the following property on the PeoplePicker:
 
 ```js
 pickerCalloutProps={{ doNotLayer: true }}
 ```
+
+#### Truncation
+By default, the PeoplePicker truncates item text in the dropdown instead of wrapping to a new line. To avoid losing meaningful information, keeping option text short is recommended, especially since localizing to different languages may increase the character count. Tooltips are not shown for truncated text within the dropdown to avoid nested popups and the usability and accessibility issues they cause.
```

Comment:
I'm not sure the bit about "keeping option text short is recommended" and loc makes sense here, since presumably the options will be names of people (where you can't control the length).

Reply:
I'll change it to "keep extra descriptive text short" and add a bit about recommending wrapping the name itself -- the truncation issues that have been raised so far have been mostly on the optional extra text after the name I think

Reply:
And IIRC wrapping the name + truncating everything else was what design recommended anyway 😄

Reply:
fixed! hope you like the new wording 😊